src/cr/cube/cube.py

```diff
@@ -7,15 +7,18 @@
 
 import copy
 import json
+from typing import FrozenSet, Iterator, List, Optional, Tuple, Union
 
 import numpy as np
 
 from cr.cube.cubepart import CubePartition
-from cr.cube.dimension import AllDimensions
-from cr.cube.enums import CUBE_MEASURE, DIMENSION_TYPE as DT, NUMERIC_MEASURES
+from cr.cube.dimension import AllDimensions, _ApparentDimensions
+from cr.cube.enums import CUBE_MEASURE
+from cr.cube.enums import DIMENSION_TYPE as DT
+from cr.cube.enums import NUMERIC_MEASURES
```

Comment:
?? isort? I do prefer the old way though

Reply:
fixed